src/bacnet/basic/tsm/tsm.c

```diff
@@ -19,10 +19,30 @@
 #include "bacnet/datalink/datalink.h"
 #include "bacnet/basic/services.h"
 #include "bacnet/basic/binding/address.h"
+#if BACNET_SEGMENTATION_ENABLED
+#include <stdlib.h>
+#include "bacnet/segmentack.h"
+#include "bacnet/abort.h"
+
+#define DEFAULT_WINDOW_SIZE 32
+
+#ifndef max
+#define max(a, b) (((a) > (b)) ? (a) : (b))
```

Comment:
There should be no need to re-define min/max since they are defined in src/bacnet/basic/sys/platform.h file.